filament/src/details/Renderer.cpp

```diff
@@ -842,7 +842,7 @@ void FRenderer::renderJob(ArenaScope& arena, FView& view) {
 
     PostProcessManager::ScreenSpaceRefConfig const ssrConfig = PostProcessManager::prepareMipmapSSR(
             fg, svp.width, svp.height,
-            ssReflectionsOptions.enabled ? TextureFormat::RGBA16F : TextureFormat::R11F_G11F_B10F,
+            (ssReflectionsOptions.enabled || view.getBlendMode() == View::BlendMode::TRANSLUCENT) ? TextureFormat::RGBA16F : TextureFormat::R11F_G11F_B10F,
```

Comment:
@pixelflinger wdyt?

Reply:
looks good to me. I'm wondering if we should, in addition, have a setting in view for this feature? So it's still possible to do refraction in a blended view in 11_11_10 -- if you know you don't care about that?

Reply:
What's your use-case for blended views? Maybe transparency in refractions would be beneficial there too.
(I'm fine with this being available either via a setting or automatically, I think I'll leave this for you to decide).